### DISCUSSION TITLE
"Add bundle item" can't find any products?

### DISCUSSION THREAD

**Discussion by aanomm** [swellstores/community#311] [Help] [answered]
Hello

Often, but not always, when I create a new Bundle product the bundle product picker doesn't show any products to select, and the search doesn't work either. The little "loading" indicator just circles endlessly?

![image](https://github.com/swellstores/community/assets/1171169/15ac5161-4a52-46ef-8db0-b4b0fb55e5f1)

**Comment by ericingram** [comment]
👀

**Comment by SusannaSwell** [comment, marked as answer]
@fucha-co Thanks for bringing it to us. I was able to reproduce this issue and let me create a bug ticket internally for faster resolution.
